userspace/libscap/engine/bpf/scap_bpf.c

```diff
@@ -908,20 +908,30 @@ static int32_t populate_syscall_routing_table_map(struct bpf_engine *handle)
 
 static int32_t populate_syscall_table_map(struct bpf_engine *handle)
 {
-	static const struct syscall_evt_pair uninterested_pair = { .flags = UF_UNINTERESTING };
 	int j;
 
 	for(j = 0; j < SYSCALL_TABLE_SIZE; ++j)
 	{
 		const struct syscall_evt_pair *p = &g_syscall_table[j];
-		if (!handle->m_syscalls_of_interest[j])
+		if(bpf_map_update_elem(handle->m_bpf_map_fds[SCAP_SYSCALL_TABLE], &j, p, BPF_ANY) != 0)
```

Comment:
Syscall table map will now always have full info about syscalls, because we will use the new interesting_syscalls_map.